diff --git a/node_modules/@vercel/nft/out/node-file-trace.js b/node_modules/@vercel/nft/out/node-file-trace.js
index 2ca08aa..b1707ea 100644
--- a/node_modules/@vercel/nft/out/node-file-trace.js
+++ b/node_modules/@vercel/nft/out/node-file-trace.js
@@ -239,7 +239,12 @@ class Job {
         if (seen.has(path))
             throw new Error('Recursive symlink detected resolving ' + path);
         seen.add(path);
-        const symlink = await this.readlink(path);
+        let symlink;
+        try {
+            symlink = await this.readlink(path);
+        } catch (e) {
+            return path;
+        }
         // emit direct symlink paths only
         if (symlink) {
             const parentPath = (0, path_1.dirname)(path);
